libraries/AP_Motors/AP_MotorsCoax.cpp

```diff
@@ -188,8 +188,19 @@ void AP_MotorsCoax::output_armed_stabilizing()
         limit.yaw = true;
     }
 
-    _thrust_yt_ccw = thrust_out + 0.5f * yaw_thrust;
-    _thrust_yt_cw = thrust_out - 0.5f * yaw_thrust;
+    // calculate the throttle setting for the lift fan
+    float thrust_yt_ccw = thrust_out + 0.5f * yaw_thrust;
+    float thrust_yt_cw = thrust_out - 0.5f * yaw_thrust;
+
+    // Apply slew limit to thrust output
+    if (!is_zero(_slew_time)) {
+        float thrust_rpyt_out_delta_max = 1.0f/(_slew_time*_loop_rate);
+        _thrust_yt_ccw += constrain_float(_thrust_yt_ccw-thrust_yt_ccw, -thrust_rpyt_out_delta_max, thrust_rpyt_out_delta_max);
+        _thrust_yt_cw += constrain_float(_thrust_yt_cw-thrust_yt_cw, -thrust_rpyt_out_delta_max, thrust_rpyt_out_delta_max);
```

Comment:
These should be `thrust_yt_ccw - _thrust_yt_ccw` and `thrust_yt_cw - _thrust_yt_cw`. If we dismiss the constrain, it's easily explained. What it is now is:
```
_thrust_yt_ccw = _thrust_yt_ccw + _thrust_yt_ccw - thrust_yt_ccw <=>
_thrust_yt_ccw = 2 * _thrust_yt_ccw - thrust_yt_ccw
```

What we want is:
```
_thrust_yt_ccw = _thrust_yt_ccw + thrust_yt_ccw - _thrust_yt_ccw <=>
_thrust_yt_ccw = thrust_yt_ccw
```

This applies to all changed motor backends.

Reply:
Yeh, I am an idiot.